vendor-extra/LiberoPageBundle/src/EventListener/SiteHeaderListener.php

```diff
@@ -24,7 +25,7 @@ public function __construct(TranslatorInterface $translator, UrlGeneratorInterfa
   public function onCreatePagePart(CreatePagePartEvent $event) : void
   {
 
-    $context = ['area' => null] + $event->getContext();
+    $context = ['area' => MAIN_GRID_FULL] + $event->getContext();
```

Comment:
This fixes https://github.com/libero/browser/pull/83#issuecomment-496581399. `area` in the context refers to the current grid area.